packages/pnpm/src/cmd/help.ts

```diff
@@ -197,26 +197,36 @@ function getHelpText (command: string) {
 
     case 'store':
       return stripIndent`
-        pnpm store status
+        pnpm store <command>
 
-        Returns a 0 exit code if packages in the store are not modified, i.e. the content of the package is the
-        same as it was at the time of unpacking.
+        Reads and performs actions on pnpm store that is on the current filesystem.
 
-        pnpm store add [pkg...]
+        Commands:
 
-        Adds new packages to the store. E.g.: pnpm store add express@4 typescript@2
+          status
 
-        pnpm store prune
+            Checks for modified packages in the store.
+            Returns exit code 0 if the content of the package is the same as it was at the time of unpacking.
 
-        Removes unreferenced (extraneous, orphan) packages from the store. Unreferenced packages are packages that are not used by
```

Comment:
Plan on updating https://pnpm.js.org with this documentation. It feels a little too bloated to be in help page of the CLI, and it makes it harder to read. Is this OK?

Reply:
I think it is ok